test/ZoraNFTCreatorV1_Fork.t.sol

```diff
@@ -178,4 +189,45 @@ contract ZoraNFTCreatorV1Test is Test, ForkHelper {
         drop.purchase{value: fee}(1);
         assertEq(drop.tokenURI(1), "DEMO");
     }
+
+    function test_ForkUpgradeWithDifferentContractName() external {
+        string[] memory forkTestChains = getForkTestChains();
+
+        for (uint256 i = 0; i < forkTestChains.length; i++) {
+            string memory chainName = forkTestChains[i];
+            vm.createSelectFork(vm.rpcUrl(chainName));
+            creator = ZoraNFTCreatorV1(getDeployment().factory);
+            verifyAddressesFork(chainName);
+            deployCore();
+            forkContractNameWithRevert();
+            forkContractNameSucceed();
+        }
+    }
+
+    function forkContractNameWithRevert() internal {
```

Comment:
could you give this a better name that describes why its reverting, like revertsWhenContractNameMismatches?